typescript/src/codeActions/custom/React/conditionalRendering.ts

```diff
@@ -0,0 +1,37 @@
+import { CodeAction } from '../../getCodeActions'
+
+/*
+    Before: <div></div>
+    After: { && (<div></div>)}
+*/
+export default {
+    id: 'conditionalRendering',
+    name: 'Wrap into Condition',
```

Comment:
```suggestion
    name: 'Wrap into Condition (&&)',
```

Reply:
Following [glean](https://marketplace.visualstudio.com/items?itemName=wix.glean) code action naming I think would be better:
```suggestion
    name: 'Render Conditionally (&&)',
```